app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/dashboard.tsx

```diff
@@ -0,0 +1,21 @@
+/** @file Main dashboard container responsible for listing user's projects as well as other
+ * interactive components. */
+
+import * as auth from "../../authentication/providers/auth";
+import withRouter from "../../navigation";
+
+// ==========================
+// === dashboardContainer ===
+// ==========================
+
+const dashboardContainer = () => {
```

Comment:
react components should use PascalCase

also not sure if `Container` suffix is convention from anywhere? what makes this the container for the dashboard, and not the dashboard itself?

Reply:
(would be great to check 3/7 through 7/7 for this if you do decide to change the naming)

Reply:
Not sure where the container suffix is from actually. It was in the first draft of the repo, from before I started working on it. I'll remove it.

Agreed on the pascal case.